### PR TITLE
login ui changes with image included and style changes in whole card layout

### PR DESCRIPTION
**Pull request by Pkjavaqwe**

### CODE REVIEW

File: src/app/components/super-admin-login/super-admin-login.component.css [hunk 1]
```diff
@@ -1,4 +1,3 @@
-html,
 div {
```
Comment (anit-neosoft) [inline src/app/components/super-admin-login/super-admin-login.component.css]: don't use direct selectors use classes

File: src/app/components/super-admin-login/super-admin-login.component.html [hunk 2]
```diff
@@ -1,50 +1,57 @@
...
-            <mat-label>Email Id</mat-label>
+<div>
+  <div class="admin-login">
+    <div class="image-section">
+      <img
+        src="https://img.lovepik.com/element/45009/2860.png_300.png"
+        alt="login image"
+      />
+    </div>
+    <div class="card">
```
Comment (anit-neosoft) [inline src/app/components/super-admin-login/super-admin-login.component.html]: There is no `mat-card` that contains the card.
refer to https://material.angular.io/components/card/examples

**Review by anit-neosoft — CHANGES_REQUESTED** [review]

File: src/app/components/super-admin-login/super-admin-login.component.css [hunk 3]
```diff
@@ -12,37 +11,71 @@ div {
   justify-content: center;
   color: rgb(19, 20, 20);
 }
+.mat-mdc-card-header {
+  justify-content: center;
+}
+.mat-icon.mat-icon.notranslate.material-icons.mat-ligature-font.mat-icon-no-color {
```
Comment (anit-neosoft) [inline src/app/components/super-admin-login/super-admin-login.component.css:17]: what is the need of this style

File: src/app/components/super-admin-login/super-admin-login.component.css [hunk 4]
```diff
@@ -12,37 +11,71 @@ div {
...
-  border-radius: 8px;
-  box-shadow: 0 8px 16px rgba(0.1, 0.1, 0.1, 0.1);
+.mat-mdc-card.admin-login {
+  display: flex;
+  flex-direction: row;
   background-color: white;
-  width: 400px;
+  width: 800px;
   height: 450px;
+  box-shadow: 0 8px 16px rgba(0.1, 0.1, 0.1, 0.1);
```
Comment (anit-neosoft) [inline src/app/components/super-admin-login/super-admin-login.component.css:28]: remove this box shadow

File: src/app/components/super-admin-login/super-admin-login.component.html [hunk 5]
```diff
@@ -1,50 +1,63 @@
-<div class="login-page">
-  <mat-card>
-    <mat-card-header>SuperAdmin Login</mat-card-header>
-    <form [formGroup]="loginForm" (submit)="loginAdmin()">
-      <div class="login">
-        <div>
-          <mat-form-field appearance="outline">
-            <mat-label>Email Id</mat-label>
+<div class="container">
```
Comment (anit-neosoft) [inline src/app/components/super-admin-login/super-admin-login.component.html:1]: use `mat-card` instead of this

Comment (anit-neosoft) [reply src/app/components/super-admin-login/super-admin-login.component.html:1]: https://material.angular.io/guide/elevation

File: src/app/components/super-admin-login/super-admin-login.component.css [hunk 6]
```diff
@@ -12,37 +11,71 @@ div {
...
+.image-section {
+  width: 400px;
+  height: 450px;
+  color: white;
+  font-weight: bold;
+  font-size: larger;
+  justify-content: center;
+}
+.image-section img {
+  width: 400px;
```
Comment (anit-neosoft) [inline src/app/components/super-admin-login/super-admin-login.component.css:57]: dont give explicit width and height
width: 100%;
  height: 100%;
  object-fit: cover;
  object-position: left;

**Review by anit-neosoft — CHANGES_REQUESTED** [review]

File: src/app/components/super-admin-login/super-admin-login.component.css (hunk 3 above)
Comment (Pkjavaqwe) [reply src/app/components/super-admin-login/super-admin-login.component.css:17]: .mat-mdc-card-header {
  justify-content: center;
}-----this, I used to make headre appear at the center as default styling to .mat-mdc-card-header is flex

**Pull request merged**